app/scripts/filters/resources.js

```diff
@@ -1274,15 +1274,36 @@ angular.module('openshiftConsole')
       });
     };
   })
-  .filter('scopeDetails', function() {
+  .filter('scopeDetails', function(sentenceCaseFilter) {
     var scopeMessages = {
       "Terminating": "Matches pods that have an active deadline.",
       "NotTerminating": "Matches pods that do not have an active deadline.",
       "BestEffort": "Matches pods that have best effort quality of service.",
       "NotBestEffort": "Matches pods that do not have best effort quality of service."
     };
     return function(scope) {
-      return scopeMessages[scope];
+      return scopeMessages[scope] || sentenceCaseFilter(scope);
+    };
+  })
+  .filter('isBestEffortPod', function(isNilFilter, usageValueFilter) {
```

Comment:
I put it in QuotaService since it relates to quota scopes, and it didn't make sense for isTerminatingPod to be in LimitsService. Would rather have them together.